Definitions for owner and controller (#137)
w3c/lws-ucs: spec/glossary.md

 - <dfn>Controller</dfn> — An entity that is capable of performing an action with a specific resource, such as updating a protected document.
 - <dfn>Entity</dfn> — An individual (person), organization, device, or process. Used interchangeably with “party”; see [NIST definition](https://csrc.nist.gov/glossary/term/entity).
 - <dfn>Identifier</dfn> — The bit string denoting the identifier associated with an Entity; see [NIST definition](https://csrc.nist.gov/glossary/term/identifier).
 - <dfn>Identity Provider</dfn> — a Service Provider that operates an Identity Service;
 - <dfn>Identity Service</dfn> — a Service that manages the subscriber’s primary authentication credentials and issues assertions derived from those credentials; see [NIST definition](https://csrc.nist.gov/glossary/term/identity_provider).
 - <dfn>Non-human Entity</dfn> — An entity with a digital identity that acts in cyberspace, but is not a human actor. This can include organizations, hardware devices, software applications, and information artifacts; see [NIST definition](https://csrc.nist.gov/glossary/term/non-human_entity)
 - <dfn>Notification Service</dfn> — TBD

>  - <dfn>Owner</dfn> — an Entity that is considered to have the rights and responsibility of a Storage.

Reviewer: ```suggestion
 - <dfn>Owner</dfn> — an Entity that is considered to have the rights and responsibility of a resource.
```
this would be more general, and aligned with the definition of controller

Reviewer 2: `the rights and responsibility of a Resource` says that the Entity is subordinate to, is restricted by, the Resource. I *think* the following communicates what is intended.
```suggestion
 - <dfn>Owner</dfn> — an Entity that is considered to have all rights and responsibilities over a Resource.
```
